source/cf/manifest.yml

```diff
@@ -0,0 +1,11 @@
+---
```

Comment:
Is there a suffix we can add to this file to indicate it's a template? Not sure if mustache has a preferred template extension but something like `manifest.yml.tpl` would be more descriptive.